pkg/eval/json_evaluator_model.go

```diff
@@ -8,6 +8,17 @@ type Flags struct {
 	Flags map[string]Flag `json:"flags"`
 }
 
+func (f Flags) Merge(ff Flags) Flags {
+	result := Flags{Flags: make(map[string]Flag)}
+	for k, v := range ff.Flags {
+		result.Flags[k] = v
+	}
+	for k, v := range f.Flags {
+		result.Flags[k] = v
+	}
+	return result
```

Comment:
I think it would be nice to have a logged warning that a flag has been overwritten when it happens, what do you folks think?